python/setup.py

```diff
@@ -59,11 +59,15 @@ class SetupType(Enum):
 
 
 class SetupSpec:
-    def __init__(self, type: SetupType, name: str, description: str):
+    def __init__(self, type: SetupType, name: str, description: str,
+                 debug_mode: bool):
         self.type: SetupType = type
         self.name: str = name
-        self.version: str = find_version("ray", "__init__.py")
+        version = find_version("ray", "__init__.py")
+        # add .dbg suffix if debug mode is on.
+        self.version: str = "%s.dbg" % version if debug_mode else version
```

Comment:
```suggestion
        self.version: str = f"{version}+dbg" if debug_mode else version
```
1. use f-string in Python to conform styles
2. needs a plus sign to conform python version scheme https://www.python.org/dev/peps/pep-0440/#local-version-segments